### PR TITLE
Bugfix/header footer

### PR DESCRIPTION
**Pull request by royderks**

### CODE REVIEW

File: src/app/components/HomePage/OrganizationCard.jsx [hunk 1]
```diff
@@ -45,15 +45,20 @@ const StyledParagraph = styled(Paragraph)`
...
     <StyledCardContent>
       <StyledHeading $as="h4" styleAs="h3">
         {shortTitle || title}
       </StyledHeading>
       <StyledParagraph>{teaser || intro}</StyledParagraph>
 
       <div>
-        <OverviewLink as={!link && RouterLink} to={to} href={link && link.uri} label="Lees meer" />
+        <OverviewLink
+          as={link && !link.uri && RouterLink}
```
Comment (chrisvanmook) [inline src/app/components/HomePage/OrganizationCard.jsx]: plz add a title attribute here :)

File: src/app/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -78,13 +65,19 @@ const Footer = ({ ...otherProps }) => (
           <Column wrap span={{ small: 1, medium: 2, big: 6, large: 10, xLarge: 10 }}>
             <FooterBottomLinkList>
               <FooterBottomLinkListItem>
-                <StyledLink to={{ type: routing.proclaimer.type }}>Privacy en cookies</StyledLink>
+                <StyledLink
+                  $as={RouterLink}
```
Comment (chrisvanmook) [inline src/app/components/Footer/Footer.jsx]: plz add title attribute here :)

File: src/app/components/Footer/FooterLinks.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { FooterLinkList, FooterLinkListItem, Link } from '@datapunt/asc-ui'
+import RouterLink from 'redux-first-router-link'
+import { toArticleDetail } from '../../../store/redux-first-router/actions'
+
+const FooterLinks = ({ children, links }) => (
+  <FooterLinkList>
+    {links &&
+      links.map(({ title, id, href, slug, order }) => {
+        const linkId = !href && id[process.env.NODE_ENV]
```
Comment (chrisvanmook) [inline src/app/components/Footer/FooterLinks.jsx]: so `id` is an array/object here? then how does it work on line 13? linkId or an array / object?! A bit confusing imho...

File: src/shared/config/config.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import cmsConfig from './cms.config'
+
+const CONTENT_LIST = require('./content-links.json')
+
+export const FOOTER_LINKS = CONTENT_LIST && CONTENT_LIST.FOOTER
```
Comment (chrisvanmook) [inline src/shared/config/config.js]: `CONTENT_LIST` is always set, right? Why not just destruct `FOOTER` `HEADER` and `NAVIGATION`?

File: src/app/components/HomePage/services/navigation-links.jsx [hunk 5]
```diff
@@ -5,13 +5,12 @@ import {
...
   toPublicationOverview,
-  toApisPage,
   toArticleDetail,
   toDatasets,
 } from '../../../../store/redux-first-router/actions'
-import { cmsIds, DATA_IN_TABLES } from '../../../../shared/config/cms.config'
+import { NAVIGATION_LINKS } from '../../../../shared/config/config'
 
-const NAVIGATION_LINKS = [
+export default [
```
Comment (aditudorache) [inline src/app/components/HomePage/services/navigation-links.jsx]: The name of the default export should match the file name. No unnamed default exports plz

File: src/shared/config/content-links.json [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+        }
+    },
+    "HELP": {
+        "id": {
+            "development": "4e5af47c-a67f-4cbc-be3c-7a0f410a45d6",
+            "production": "de7d3101-354c-44f6-b94d-dde572239090"
+        },
+        "slug": "help"
+    }
+}
```
Comment (aditudorache) [inline src/shared/config/content-links.json]: newline here?

**Pull request merged**